app/views/reviews/new.html.erb

```diff
@@ -13,5 +13,5 @@
 
 <% end %>
 
-<%= link_to "Back to Homepage", parks_path %>
+<%= link_to "Back to #{@park.name}", park_path(@park) %>
```

Comment:
I think this is a regression? If this is the new page, you don't want a user to go back to the show page (for the non-existent park)